assets/styles/layouts/partials/speaker-avatar.css

```diff
@@ -15,7 +15,6 @@
     opacity: 0.5;
     transition: border 0.3s ease;
     position: relative;
-    z-index: 1;
```

Comment:
Could you please double-check if the hover works as expected?

Reply:
Sure (pointer not visible on screenshot):

<img width="369" alt="image" src="https://github.com/user-attachments/assets/6509aa75-567d-4f79-8a93-5d360f4de45c">